galite-core/src/main/kotlin/org/kopi/galite/report/VReport.kt

```diff
@@ -15,6 +15,612 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+
 package org.kopi.galite.report
 
-abstract class VReport 
\ No newline at end of file
+import java.io.File
+import java.net.MalformedURLException
+import java.text.MessageFormat
+import java.util.Locale
+import java.util.Vector
+
+import org.kopi.galite.chart.VHelpGenerator
```

Comment:
why chart/VHelpGenerator? report have it's own help generator

Reply:
Fixed